Colours_And_Fonts_style_v1.css

```diff
@@ -128,31 +194,55 @@ ul li:hover ul {
 
 .box1 {
   width: 1990px;
-  height: 20px;
-  background-color: #BC1327;
+  height: 10px;
+  background-color: #A2091B;
 }
 
 .box2 {
 	width: 1990px;
-	height: 20px;
-	background-color: #ffdd00;
+	height: 10px;
+	background-color: #FBCA01;

```

Comment:
I changed the height of from 20px to 10px as the height was too big. I also changed the colour from drak yellow to a lighter yellow to match the colours of Tawa College.